workspaces/meewee-time-registration/plugins/plugin-meewee-time-registration/meewee-time-registration/README.md

```diff
@@ -0,0 +1,36 @@
+# MeeWee Time Registration Plugin
+
+This plugin is contributed by [Dewise](https://www.dewise.com/)
+
+## Features
```

Comment:
Screen shots are really helpful, you have some in the PR description that would be great to add here!

Reply:
Done 👍